draft-ietf-quic-tls.md

```diff
@@ -1503,42 +1503,54 @@ SHOULD track redundant packets and treat excessive volumes of any non-productive
 packets as indicative of an attack.
 
 
-# Error codes {#errors}
+# Error Codes {#errors}
 
-The portion of the QUIC error code space allocated for the crypto handshake is
-0xC0000000-0xFFFFFFFF. The following error codes are defined when TLS is used
-for the crypto handshake:
+This document defines error codes from the error code space used in
+{{QUIC-TRANSPORT}}.
```

Comment:
The crypto handshake bit is on the very next line.  The point of separation is that the codes are consumed even if TLS isn't used.